src/modules/dashboard/components/ExpenditurePage/ExpenditureSettings/Streaming/LockedFundingSource/LockedFundingSource.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useCallback, useMemo } from 'react';
+import classNames from 'classnames';
+import { defineMessages, useIntl } from 'react-intl';
+import { ROOT_DOMAIN_ID } from '@colony/colony-js';
+
+import ColorTag, { Color } from '~core/ColorTag';
+import { FormSection, InputLabel } from '~core/Fields';
+import TokenIcon from '~dashboard/HookedTokenIcon';
+import { Colony } from '~data/index';
+import Numeral from '~core/Numeral';
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+
+import { FundingSource } from '../types';
+
+import styles from './LockedFundingSource.css';
+
+const MSG = defineMessages({
+  team: {
+    id: 'dashboard.ExpenditurePage.Streaming.LockedFundingSource.team',
+    defaultMessage: 'Team',
+  },
+  rate: {
+    id: 'dashboard.ExpenditurePage.Streaming.LockedFundingSource.rate',
+    defaultMessage: 'Rate {count}',
+  },
+  limit: {
+    id: 'dashboard.ExpenditurePage.Streaming.LockedFundingSource.limit',
+    defaultMessage: 'Limit {count}',
+  },
+});
+
+const displayName = 'dashboard.ExpenditurePage.Streaming.LockedFundingSource';
+
+interface Props {
+  fundingSource?: FundingSource;
+  colony: Colony;
+  isOpen?: boolean;
+}
+
+const LockedFundingSource = ({ fundingSource, colony, isOpen }: Props) => {
+  const { formatMessage } = useIntl();
+  const { rate, team } = fundingSource || {};
+
+  const domain = useMemo(
+    () =>
+      colony?.domains.find(({ ethDomainId }) => Number(team) === ethDomainId),
+    [colony, team],
+  );
+
+  const getDomainColor = useCallback<(domainId: string | undefined) => Color>(
+    (domainId) => {
+      const rootDomainColor: Color = Color.LightPink;
+      const defaultColor: Color = Color.Yellow;
+      if (domainId === String(ROOT_DOMAIN_ID)) {
+        return rootDomainColor;
+      }
+      if (!colony || !domainId) {
+        return defaultColor;
+      }
+      return domain ? domain.color : defaultColor;
+    },
+    [colony, domain],
+  );
+
+  return (
+    <>
+      {isOpen && (
```

Comment:
If this condition wraps all the elements, then maybe you could do this check a few lines earlier:
```
if(!isOpen) {
  return null;
}
```
And have the elements nested by 1 level less.